src/js/components/CheckBoxGroup/CheckBoxGroup.js

```diff
@@ -60,7 +62,17 @@ export const CheckBoxGroup = forwardRef(
     };
 
     return (
-      <Box ref={ref} gap={gap} {...rest}>
+      <StyledCheckBoxGroup
+        ref={ref}
+        {...theme.checkBoxGroup.container}
+        gap={
+          gap ||
+          (theme.checkBoxGroup.container && theme.checkBoxGroup.container.gap
```

Comment:
What happens if a caller has set `theme.checkBoxGroup` to `undefined`?

Reply:
I set `checkBoxGroup` as defined in base.js and I believe `Grommet` component merges any custom theme with `base.js` so it will always be defined.